components/automate-ui/src/app/pages/+compliance/+node-credentials/node-credentials-list/node-credential-list.component.html

```diff
@@ -0,0 +1,67 @@
+<div class="content-container">
+  <div class="container">
+    <main>
+      <chef-page-header>
+        <chef-heading>Node Credentials</chef-heading>
+        <chef-subheading>
+          SSH, WinRM, and sudo credentials to remotely access nodes.
+        </chef-subheading>
+      </chef-page-header>
+      <app-create-node-credential-modal
+        [openEvent]="openUserModal"
+        [sortParams]="params"></app-create-node-credential-modal>
+      <app-delete-object-modal
+        [visible]="deleteModalVisible"
+        objectNoun="node Credential"
+        [objectName]="nodeCredentialToDelete?.name"
+        [moreDetails]="inUseMessage()"
+        (close)="closeDeleteModal()"
+        (deleteClicked)="deleteNodeCredential()"
+        objectAction="Delete">
+      </app-delete-object-modal>
+      <section class="page-body">
+        <chef-loading-spinner *ngIf="nodesListLoading" size="50"></chef-loading-spinner>
+        <ng-container *ngIf="!nodesListLoading">
+          <chef-toolbar>
+            <chef-button primary (click)="openCreateModal()" data-cy="team-create-button">Create Credential
+            </chef-button>
+          </chef-toolbar>
```

Comment:
Would like to wrap in an authorization guard. Are you familiar with this notion? If not, search through the code base for other examples to see how they are used.
```suggestion
        <app-authorized [allOf]="['/api/v0/secrets', 'post']">
          <chef-toolbar>
            <chef-button primary (click)="openCreateModal()" data-cy="team-create-button">Create Credential
            </chef-button>
          </chef-toolbar>
        </app-authorized>
```